README.md

```diff
@@ -73,6 +73,17 @@ using KubeFed.
 Take a look at our [development guide](docs/development.md) if you are
 interested in contributing.
 
+## Community
+
+Refer [contributing guidelines](./CONTRIBUTING.md) before contributing to KubeFed.
+
+### Communication channels
+
+KubeFed is part of [multicluster SIG](https://github.com/kubernetes/community/tree/master/sig-multicluster) and it uses the same communication channels as multicluster SIG.
```

Comment:
s/is part of multicluster SIG and it/is sponsored by SIG Multicluster and/

s/as multicluster SIG//